src/models/badge.rs

```diff
@@ -68,6 +68,10 @@ pub enum Badge {
         branch: Option<String>,
         service: Option<String>,
     },
+    BitbucketPipelines {
+        repository: String,
+        branch: String,
```

Comment:
I’ve made the branch attribute required for now, because while Shields.io [allows the branch to be omitted](https://shields.io/category/build), it [assumes the default branch is “master”](https://github.com/badges/shields/blob/436d2ba3a03e2464a2cd5198b811dc14ede26bec/services/bitbucket/bitbucket-pipelines.service.js#L102), but Bitbucket lets you choose any branch as the default branch

Reply:
(strictly speaking, this is a bug that Shields.io could fix by [querying the Bitbucket API](https://api.bitbucket.org/2.0/repositories/delan/iyypdgarimwgp) for the repository’s main branch, but that’s a job for another day)